data/cameras.xml

```diff
@@ -17291,7 +17291,9 @@
 		<Sensor black="256" white="62914"/>
 		<Aliases>
 			<Alias id="X1D II 50C">Hasselblad X1D II 50C</Alias>
-			<Alias id="X1DM2-50c">Hasselblad X1DM2-50c</Alias>
+			<Alias id="X1D II 50C">Hasselblad X1DM2-50c</Alias>
+			<Alias id="CFV II 50C">Hasselblad CFV II 50C</Alias>
+			<Alias id="CFV II 50C">CFV II 50C/907X</Alias>
```

Comment:
Not `<Alias id="CFV II 50C">Hasselblad CFV II 50C/907X</Alias>` ?
(sample missing, but i'll ignore that here.)

Reply:
It really is just `CFV II 50C/907X` in the Exif `Model` string in [the DPR sample gallery](https://www.dpreview.com/sample-galleries/0891752730/hasselblad-907x-and-cfv-ii-50c-sample-gallery).

Presumably, when the .3FR gets munged in tethering by Hasselblad's Phocus software to an .FFF, this becomes `Hasselblad CFV II 50C` (also confirmed in Exif in JPEG samples on the Hasselblad's web site), as we've also seen w/ e.g. the X2D 100C RPU samples.

Reply:
Also note that the CVF 100C was released yesterday, but I won't be adding aliases for those until some samples appear...